universal-application-tool-0.0.1/test/FunctionalTest.java

```diff
@@ -60,14 +61,30 @@ public void listPersons() {
   }
 
   @Test
-  public void addPerson() {
+  public void createPerson() {
     Http.RequestBuilder request =
-        fakeRequest(routes.PostgresController.add("John"))
+        fakeRequest(routes.PostgresController.create("John"))
             .header(Http.HeaderNames.HOST, "localhost:" + testServerPort());
     Result result = route(app, request);
 
     assertThat(result.status()).isEqualTo(OK);
-    assertThat(contentAsString(result)).contains("person John with ID:", "added.");
+    assertThat(contentAsString(result)).contains("person John with ID:", "created.");
+  }
+
+  @Test
+  public void createPersonSynchronously() {
+    final int oldNumRecord = Ebean.find(Person.class).findCount();
+
+    Http.RequestBuilder request =
+        fakeRequest(routes.PostgresController.createSync("John"))
+            .header(Http.HeaderNames.HOST, "localhost:" + testServerPort());
+    Result result = route(app, request);
+
+    assertThat(result.status()).isEqualTo(OK);
+    assertThat(contentAsString(result)).contains("person John with ID:", "synchronously created.");
+
+    final int newNumRecord = Ebean.find(Person.class).findCount();
```

Comment:
I wonder if this can just be

```suggestion
    final int newNumRecord = new Person().findCount();
```

Reply:
This unfortunately doesn't work. `findCount()` is a member of the query beans class[1], not `Person` class.

I cannot access autogenerated query beans class `QPerson` in Play for some reason.

[1] https://ebean.io/docs/query/query-beans

Reply:
Ah that's a bummer. Yeah, I found this PR which makes it look like it's been quite the pain point for the framework developers: https://github.com/playframework/play-ebean/pull/64